app/controllers/application_controller.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::API
+  include DeviseTokenAuth::Concerns::SetUserByToken
```

Comment:
let's create a ApiController which inherits from Action Controller::Api. That should be placed in the app/controllers directory. Then add this line to it.